index.js

```diff
@@ -1,15 +1,24 @@
-module.exports = {
-  presets: [
-    require('babel-preset-es2015-without-strict'),
-    require('babel-preset-react'),
-  ],
-  plugins: [
-    [require('babel-plugin-transform-es2015-template-literals'), { spec: true }],
-    require('babel-plugin-transform-es3-member-expression-literals'),
-    require('babel-plugin-transform-es3-property-literals'),
-    require('babel-plugin-transform-jscript'),
-    require('babel-plugin-transform-exponentiation-operator'),
-    require('babel-plugin-syntax-trailing-function-commas'),
-    [require('babel-plugin-transform-object-rest-spread'), { useBuiltIns: true }],
-  ],
-};
+module.exports = function(context, options) {
+  var es2015preset;
+  if (options && options.modules === false) {
+    es2015preset = ['es2015', { modules: false }];
```

Comment:
whoops, i read your note.

given that, i think we need to make `babel-preset-es2015` a peer dep?

If that's not necessary - if the string works - then can we use the string in all cases in this file, instead of `require`?